pliers/extractors/text.py

```diff
@@ -402,3 +400,47 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+
+class WordCounterExtractor(BatchTransformerMixin, TextExtractor):
+
+    ''' Extracts number of times each unique word has occurred within text
+    
+    Args:
+        lemmatize(bool): specifies if words are to be lemmatized before being counted. 
```

Comment:
This probably doesn't need to be here. @rbroc mentioned she wrote a fitler to do this, so that's a better place for it.